### ISSUE TITLE
Pokemon Colosseum and XD support

### ISSUE THREAD

**Issue by StarsMmd** [Ploaj/HSDLib#24]
Hi there. So Pokemon Colosseum and XD use the same file format for their models. We've been using this tool to successfully export models from the game but are having trouble with reimporting them. I don't know how much of this is due to slight differences between the models between the games and/or how much is due to us just not using the tool properly.

Would you be willing to officially support compatibility with Colo/XD as well? We have a pretty dedicated RE team with members that would be able to contribute as well.

**Comment by Ploaj** [comment]
Do you all have a discord or something?

**Comment by trevor403** [comment]
Yes, it is called NGC/Wii Pokemon ROM hacking https://discord.gg/pE4uSx
